view/src/components/event.js

```diff
@@ -343,6 +367,100 @@ class event extends Component {
               </Card>
             </Grid>
           </Grid>
+          <Grid>
+        <br/>
+
+        <Grid/>
+        { this.state.netid == 'swe123456'? 
+        (<Grid 
+        container
+        spacing={2}
+        height="100%"
+        width="100%"
+        alignItems="stretch"
+        justify="space-evenly">
+          <Grid item xs={12}>
+        <Card
+        height="100%"
+        className="movingItem"
+        variant="outlined"
+        style={{ padding: "10px" }}
+      >
+          <CardContent height="100%" align="center">
+                  
+                  <h1 align="center">Member List</h1><Divider />
+                  <SearchPage/>
+                  <br />
+                  {this.state.users == null ? (
+            <h4>No users are currently on the Member List.</h4>
+          ) : (
+            /* this.state.users
```

Comment:
can this be removed

Reply:
Yeah it can be, I just put it there for debugging purposes.